src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -611,13 +611,21 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt, $
             }
 
             if ($token->equals('(')) {
-                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+                $end = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+
+                if (preg_match('/(?<!\$)\bfunction\b/', $tokens->generatePartialCode($index, $end)) !== 1) {
```

Comment:
I'm pretty sure this will match comments like e.g. `/* function */`. Again, checking what the generated code looks like is not a good approach IMO. Check the kind of tokens instead (`T_COMMENT`, `T_WHITESPACE` and so on).